### PR TITLE
All notes

### PR DESCRIPTION
**Pull request by pankaj-prog**
notes page screen: 
all notes

### CODE REVIEW

File: src/components/IconButton/IconButton.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const IconButton = ({ icon, name, clickHandler }) => {
+  return (
+    <div className="icon-btn-wrapper">
+      <button className="btn text-lg" onClick={(e) => clickHandler(e)}>
```
Comment (tarunsankhla) [inline src/components/IconButton/IconButton.jsx]: Directly do {clickhandler}

File: src/components/NoteListSection/NoteListSection.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+const NoteListSection = ({ notesList }) => {
+  return (
+    <section className="note-list-section">
+      {/* add options for sort and filter here */}
```
Comment (tarunsankhla) [inline src/components/NoteListSection/NoteListSection.jsx:6]: Remove comments

File: src/index.css [hunk 3]
```diff
@@ -60,3 +62,50 @@ body {
...
+  top: 3rem;
+  right: 0;
+  min-width: 100%;
+  padding: 0 4px;
+  text-align: center;
+  transition: opacity 0.6s ease;
+}
+
+.icon-btn-wrapper:hover .name {
+  opacity: 1;
```
Comment (tarunsankhla) [inline src/index.css:84]: What are you trying to do here default opacity is 1

File: src/styles/NotesColumn.css [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+.note-list-section {
+  flex-grow: 1;
+  overflow-y: scroll;
+}
+
+.note-list-section::-webkit-scrollbar,
+.ql-editor::-webkit-scrollbar {
+  width: 10px;
+  background-color: var(--COLOR-SECONDARY-V2);
+  /* border: 1px solid var(--COLOR-SECONDARY-V3); */
```
Comment (tarunsankhla) [inline src/styles/NotesColumn.css]: Remove comments

**Review by tarunsankhla — APPROVED** [review]
Over all great make the UI a little color full or add a action color 
Rest everything looks great

File: src/components/Sidebar/Sidebar.jsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    <nav className="sidebar">
+      <ul>
+        {links.map((link) => {
+          return (
+            <li>
+              <NavLink to={link.pathName}>{link.name}</NavLink>
+            </li>
+          );
+        })}
+      </ul>
```
Comment (PuneetP16) [inline src/components/Sidebar/Sidebar.jsx:21]: Appreciation: Nice you have mapped NavLink.

**Review by PuneetP16 — APPROVED** [review]
UI Appreciation: Your UI looks great. Good to merge.

**Review by SinchanaMS — APPROVED** [review]
Everything looks great!
I liked how you've mapped over the NavLinks and the absolute imports. 
The UI looks great too! Good work!

File: src/index.css (hunk 3 above)
Comment (pankaj-prog) [reply src/index.css:84]: initially, I have kept opacity as 0 and on hovering I am making it 1

**Pull request merged**